app/controllers/records_controller.rb

```diff
@@ -1,8 +1,22 @@
 class RecordsController < ApplicationController
   include RecordsControllerBehavior
 
+  def ingest_options
+    @templates = [["Raw (no template)", nil]] + template_class.all.collect {|t| [t.title, t.id]}
+  end
+
   protected
 
+  def build_form
```

Comment:
I vote for the second one. It looks pretty and thats the only reason I can really think of. I cant really see any perks of using one over the other.